src/driver/mysql/MysqlDriver.ts

```diff
@@ -907,6 +916,11 @@ export class MysqlDriver implements Driver {
      * Loads all driver dependencies.
      */
     protected loadDependencies(): void {
+        if (this.options.driver === "mysql2") {
```

Comment:
why this code block is needed? You already set `usingMySQL2` down below.

Reply:
because I didn't find a way to select driver explicitly, for unit test, Unless I uninstall the `mysql` package.

Reply:
the same applies versa. If you have mysql2 installed (or both) and you want to test `mysql` explicitly - you can't.

Reply:
(because of the lines you added)

Reply:
sorry, I can't get your point. My idea is, If a connection is specified with `mysql2`, it will use it. Otherwise, it keeps use `mysql` first. And obviously you can setup two connection to test both `mysql` and `mysql2`.

Reply:
ah okay, I misunderstood your code

Reply:
Anyway, I think this is redundant for now. I would like to keep current loading behaviour as it is now.